rss_reader/args_parser.py

```diff
@@ -0,0 +1,72 @@
+"""
+This module process the input arguments
+"""
+
+import argparse
+import re
+import rss_reader.logs as logs
```

Comment:
It is better to use `from <package> import module` here

Reply:
fixed